feature/experimental/p4rt/ate_tests/traceroute_packetout_test/packetout_test.go

```diff
@@ -62,86 +62,63 @@ func sendPackets(t *testing.T, client *p4rt_client.P4RTClient, packets []*p4v1.P
 func testPacketOut(ctx context.Context, t *testing.T, args *testArgs) {
 	leader := args.leader
 	desc := "PacketOut from Primary Controller"
-	ttls := []int{0, 1}
+	ttl := 1
 	//for ipv4
 	t.Run(desc+" ipv4 ", func(t *testing.T) {
 		// Check initial packet counters
 		port := sortPorts(args.ate.Ports())[0].Name()
+		t.Logf("Sending ipv4 pakcets with ttl %d", ttl)
+		counter0 := gnmi.Get(t, args.ate, gnmi.OC().Interface(port).Counters().InPkts().State())
+		t.Logf("Initial number of packets: %d", counter0)
 
-		for _, ttl := range ttls {
-			t.Logf("Sending ipv4 pakcets with ttl %d", ttl)
-			counter0 := gnmi.Get(t, args.ate, gnmi.OC().Interface(port).Counters().InPkts().State())
-			t.Logf("Initial number of packets: %d", counter0)
+		packets := args.packetIO.GetPacketOut(portId, true, uint8(ttl))
+		packetCounter := 100
+		t.Logf("Sending packets now")
 
-			packets := args.packetIO.GetPacketOut(portId, true, uint8(ttl))
-			packetCounter := 100
-			t.Logf("Sending packets now")
+		sendPackets(t, leader, packets, packetCounter)
 
-			sendPackets(t, leader, packets, packetCounter)
+		// Wait for ate stats to be populated
+		time.Sleep(60 * time.Second)
 
-			// Wait for ate stats to be populated
-			time.Sleep(60 * time.Second)
+		// Check packet counters after packet out
+		counter1 := gnmi.Get(t, args.ate, gnmi.OC().Interface(port).Counters().InPkts().State())
+		t.Logf("Final number of packets: %d", counter1)
 
-			// Check packet counters after packet out
-			counter1 := gnmi.Get(t, args.ate, gnmi.OC().Interface(port).Counters().InPkts().State())
-			t.Logf("Final number of packets: %d", counter1)
+		// Verify InPkts stats to check P4RT stream
+		t.Logf("Received %v packets on ATE port %s", counter1-counter0, port)
 
-			// Verify InPkts stats to check P4RT stream
-			t.Logf("Received %v packets on ATE port %s", counter1-counter0, port)
-
-			if ttl == 0 {
-				if counter1-counter0 > uint64(float64(packetCounter)*0.05) {
-					t.Fatalf("Ttl=0 packets are being forwarded.")
-				}
-			} else {
-				if counter1-counter0 < uint64(float64(packetCounter)*0.95) {
-					t.Fatalf("Not all the packets are received.")
-				}
-			}
-			time.Sleep(20 * time.Second)
+		if counter1-counter0 < uint64(float64(packetCounter)*0.95) {
```

Comment:
nit: Can you lift the `0.95` up to the top of the file and declare it as a variable say `allowCounterDiff = 0.95` and use it in all the places. This helps in debugging later.